### PR TITLE
Add details to the store and node pages.

### PR DESCRIPTION
**Pull request by BramGruneir**

### CODE REVIEW

File: resource/ui/ts/models/node_status.ts [hunk 1]
```diff
@@ -88,6 +93,56 @@ module Models {
...
+            private _replicated(nodeId: string): string {
+                var node = this.statuses()[nodeId];
+                if (node.leader_range_count == 0) {
+                    return "100%";
+                }
+                return (node.replicated_range_count / node.leader_range_count * 100).toString() + "%";
+            }
+
+            // TODO(Bram): Move to utility class.
+            public static _datetimeFormater = d3.time.format("%Y-%m-%d %H:%M:%S")
```
Comment (mrtracy) [inline resource/ui/ts/models/node_status.ts]: private?

File: resource/ui/ts/models/node_status.ts [hunk 2]
```diff
@@ -88,6 +93,56 @@ module Models {
...
+            private static _convertDate(nanos: number): string {
```
Comment (mrtracy) [inline resource/ui/ts/models/node_status.ts]: Not really a conversion, just formatting as a string.

File: resource/ui/ts/models/store_status.ts [hunk 3]
```diff
@@ -89,6 +95,50 @@ module Models {
...
+            public static _datetimeFormater = d3.time.format("%Y-%m-%d %H:%M:%S")
+            private static _convertDate(nanos:number):string {
+                var datetime = new Date(nanos / 1.0e6);
+                return Stores._datetimeFormater(datetime);
+            }
+
+            public Details(storeId:string):_mithril.MithrilVirtualElement {
+                var store = this.statuses()[storeId];
+                return m("div",[
+                    m("table", [
```
Comment (mrtracy) [inline resource/ui/ts/models/store_status.ts]: Your use of tables to display one column sickens me.

Comment (mrtracy) [inline resource/ui/ts/models/store_status.ts]: I guess its fine, just make it look good.

**Comment by mrtracy** [comment]
LGTM

File: resource/ui/ts/models/node_status.ts (hunk 2 above)
Comment (BramGruneir) [inline resource/ui/ts/models/node_status.ts]: Good call.  Renamed to _formatDate.

File: resource/ui/ts/models/store_status.ts (hunk 3 above)
Comment (BramGruneir) [inline resource/ui/ts/models/store_status.ts]: (╯°□°)╯︵ ┻━┻

File: resource/ui/ts/models/node_status.ts (hunk 1 above)
Comment (BramGruneir) [inline resource/ui/ts/models/node_status.ts]: fixed.

**Pull request merged**